packages/keyring-snap-bridge/src/SnapKeyring.ts

```diff
@@ -349,6 +352,48 @@ export class SnapKeyring extends EventEmitter {
     return null;
   }
 
+  /**
+   * Handle a balances updated event from a Snap.
+   *
+   * @param message - Event message.
+   * @returns `null`.
+   */
+  async #handleAccountBalancesUpdated(message: SnapMessage): Promise<null> {
+    assert(message, AccountBalancesUpdatedEventStruct);
+    const event = message.params;
+
+    this.#messenger.publish('SnapKeyring:accountBalancesUpdated', event);
+    return null;
+  }
+
+  /**
+   * Handle a asset list updated event from a Snap.
+   *
+   * @param message - Event message.
+   * @returns `null`.
+   */
+  async #handleAccountAssetListUpdated(message: SnapMessage): Promise<null> {
+    assert(message, AccountAssetListUpdatedEventStruct);
+    const event = message.params;
+
+    this.#messenger.publish('SnapKeyring:accountAssetListUpdated', event);
+    return null;
+  }
+
+  /**
+   * Handle a transactions updated event from a Snap.
+   *
+   * @param message - Event message.
+   * @returns `null`.
+   */
+  async #handleAccountTransactionsUpdated(message: SnapMessage): Promise<null> {
+    assert(message, AccountTransactionsUpdatedEventStruct);
+    const event = message.params;
+
+    this.#messenger.publish('SnapKeyring:accountTransactionsUpdated', event);
+    return null;
```

Comment:
I wonder if it's worth to refactor these handlers to reuse the same code since only the struct and event name are different. WDYT?

Reply:
Yup, I kinda thought about the same TBH 😄 I'll do that!